plugins/PostProcessingPlugin/scripts/MultiExtColorMix.py

```diff
@@ -0,0 +1,623 @@
+                    "enabled": "invis_setting"
+                },
+                "T2_mix_start":
+                {
+                    "label": "    Ext-2 Start mix %",
+                    "description": "Third extruder percentage 0-100",
+                    "type": "int",
+                    "unit": "%",
+                    "default_value": 0,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "0",
+                    "maximum_value_warning": "100",
+                    "enabled": "T2_include"
+                },
+                "T2_mix_end":
+                {
+                    "label": "    Ext-2 End mix %",
+                    "description": "Third extruder percentage 0-100 to finish blend",
+                    "type": "int",
+                    "unit": "%",
+                    "default_value": 100,
+                    "minimum_value": "0",
+                    "minimum_value_warning": "0",
+                    "maximum_value_warning": "100",
+                    "enabled": "T2_include and mix_style == 'gradient'"
+                },
+                "park_head":
+                {
+                    "label": "Park and Purge",
+                    "description": "Whether to Park the head and Purge before starting a new Mix Ratio.",
+                    "type": "bool",
+                    "default_value": false,
+                    "enabled": true
+                },
+                "park_x":
+                {
+                    "label": "    Park X Location",
+                    "description": "What X.",
+                    "type": "int",
+                    "default_value": 0,
+                    "enabled": "park_head"
+                },
+                "park_y":
+                {
+                    "label": "    Park Y Location",
+                    "description": "What Y.",
+                    "type": "int",
+                    "default_value": "0",
+                    "enabled": "park_head"
+                },
+                "purge_amt":
+                {
+                    "label": "    Purge Amount",
+                    "description": "The extruder to use after mixing ends.  The mixing extruder is 'Extruder_Count +1 (will be T3 in the gcode).",
+                    "type": "int",
+                    "unit": "mm ",
+                    "default_value": 25,
+                    "minimum_value": "1",
+                    "maximum_value": "50",
+                    "enabled": "park_head"
+                }
+            }
+        }"""
+    
+    def execute(self, data):
+        MyCura = Application.getInstance().getGlobalContainerStack()   
+        extruder_count = MyCura.getProperty("machine_extruder_count", "value")
+        print_sequence = str(MyCura.getProperty("print_sequence", "value"))
+        init_extruder = Application.getInstance().getExtruderManager().getInitialExtruderNr()
+        if extruder_count >1:
```

Comment:
small formatting mistake
```suggestion
        if extruder_count > 1:
```